airflow/api/client/local_client.py

```diff
@@ -22,21 +22,39 @@
 from airflow.api.common import delete_dag, trigger_dag
 from airflow.api.common.experimental.get_lineage import get_lineage as get_lineage_api
 from airflow.exceptions import AirflowBadRequest, PoolNotFound
+from airflow.models import DagRun
 from airflow.models.pool import Pool
 
 
 class Client(api_client.Client):
     """Local API client implementation."""
 
-    def trigger_dag(self, dag_id, run_id=None, conf=None, execution_date=None, replace_microseconds=True):
+    def trigger_dag(
+        self, dag_id, run_id=None, conf=None, execution_date=None, replace_microseconds=True
+    ) -> dict | None:
         dag_run = trigger_dag.trigger_dag(
             dag_id=dag_id,
             run_id=run_id,
             conf=conf,
             execution_date=execution_date,
             replace_microseconds=replace_microseconds,
         )
-        return f"Created {dag_run}"
+        if isinstance(dag_run, DagRun):
```

Comment:
```suggestion
        if dag_run:
```
Do we need the isinstance?
Also let's add a test for this change in tests/api/client/test_local_client.py